src/components/profile/profile.tsx

```diff
@@ -49,6 +51,17 @@ class Profile extends React.Component<Props> {
         if (!this.props.currentState) {
             this.props.getStateByPublicKey(this.props.currentUser.publicKey);
         }
+
+        // @ts-ignore
```

Comment:
What's with the ignore?

Reply:
TSLint thinks the `setInterval` function returns type `Timeout` but it actually returns `number`.  If the the the timerId type is changed to `Timeout` the code won't compile.